dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/SchedulerServiceImpl.java

```diff
@@ -171,6 +171,9 @@ public Map<String, Object> insertSchedule(User loginUser,
         scheduleObj.setProcessDefinitionName(processDefinition.getName());
 
         ScheduleParam scheduleParam = JSONUtils.parseObject(schedule, ScheduleParam.class);
+        if (now.after(scheduleParam.getStartTime())) {
+            scheduleParam.setStartTime(now);
```

Comment:
It's better don't modify the user's input, this may be thought as a bug.

Reply:
> It's better don't modify the user's input, this may be thought as a bug.

Okay, i'll make some changes that return warning message if setting start time earlier than current time, also change the default start time in front-end to 0:00 of the next day? WDYT.

Reply:
Can we rollback the strategy?

Reply:
How to handle the scenario described in https://github.com/apache/dolphinscheduler/issues/12231?

Reply:
@ruanwenjun Any new suggestions?